pangeo-notebook/packages.txt

```diff
@@ -425,7 +423,7 @@ overrides==7.4.0
 p11-kit==0.24.1
 packaging==23.1
 pamela==1.1.0
-pandas==2.1.0
```

Comment:
not sure why pandas would go from 2->1

Reply:
erddapy-0.7.2 uses `pandas >=0.20.3,<2a0` :upside_down_face: